Name: dsb-phn-snds
Version: 2
Description: disable end call, dropped call, missed call, ignored call sounds. The missed call will now just appear like a dashboard notification.

--- .orig/usr/palm/applications/com.palm.app.phone/app/controllers/announcer-assistant.js
+++ /usr/palm/applications/com.palm.app.phone/app/controllers/announcer-assistant.js
@@ -322,7 +322,7 @@
 					"height": height, 
 				}, 
 				this.onAnnounceMissedCreated.bind(this, name, number, picture, contact, Messages.missedMessage, timeStamp),
-				'popupalert');
+				'dashboard');
 		}
 
 	},
@@ -439,7 +439,7 @@
 		this.appControl.showBanner(
 			{
 				"messageText": message,
-				"soundClass": 'notifications'
+				"soundClass": ''
 			}, 
 			
 			{
@@ -770,7 +770,7 @@
 			this.appControl.showBanner(
 				{
 					messageText: message,
-					soundClass: 'notifications'
+					soundClass: ''
 				}, 
 				{
 					prefill: callState.number
@@ -864,7 +864,7 @@
 		this.appControl.showBanner(
 			{
 				messageText: message,
-				soundClass: 'notifications'
+				soundClass: ''
 			}, 
 			{
 				prefill: callbackNumber
